srv/modules/runners/ready.py

```diff
@@ -72,7 +72,7 @@ def apparmor(self):
         """
         contents = self.local.cmd(self.search , 'cmd.shell', [ '/usr/sbin/apparmor_status --enabled 2>/dev/null; echo $?' ], expr_form="compound")
         for minion in contents:
-            if contents[minion] == 0:
+            if contents[minion] == '0':
```

Comment:
I get the feeling you've been bitten before, Joshua :)

Sure, I'll update the PR.  Thanks for the input.